spec/feature_spec_helper.rb

```diff
@@ -0,0 +1 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.